libs/island-ui/core/src/lib/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { CSSProperties, forwardRef, ReactNode } from 'react'
+
+const style: CSSProperties = {
+  border: 0,
+  clip: 'rect(0 0 0 0)',
+  height: '1px',
+  overflow: 'hidden',
+  padding: 0,
+  position: 'absolute',
+  whiteSpace: 'nowrap',
+  width: '1px',
+}
+
+interface VisuallyHiddenProps
+  extends Pick<
+    React.HTMLAttributes<HTMLElement>,
+    'aria-labelledby' | 'aria-describedby' | 'id'
+  > {
+  children?: ReactNode
+}
+
+const VisuallyHidden = forwardRef<HTMLSpanElement, VisuallyHiddenProps>(
+  function VisuallyHiddenInner({ children, ...restProps }, ref) {
+    return (
```

Comment:
Why use inline styles rather than a className and vanilla extract?

Reply:
Both ways have their minor pros and cons, so I had to choose either way. I chose inline because this kind of component is usually written with inline styles, so that felt more natural.